### ISSUE TITLE
AI Android failure highlight color changes in high contrast mode

### ISSUE THREAD

**Issue by pownkel** [microsoft/accessibility-insights-web#3694]
## Describe the bug

In Accessibility Insights for Android, the red color highlighting failure instances in the app screenshot changes in high contrast mode to match the lighter red that we use against the black background. It seems odd to change the highlight color on a non-changing background, particularly when this behavior doesn't happen in web.

Also, although we can't predict the background color of the screenshot and guarantee contrast, the lighter shade of red fails a contrast check against the white background that most of the default android apps use.

**To Reproduce**

1. Open Accessibility Insights for Android and connect a device (or use mock-adb)
2. Scan an app that has accessibility issues
3. Go to Settings and turn on high contrast mode
4. Notice that the red "!" icons and highlights on the screenshot have become lighter

## Expected behavior

The red highlight color stays the same in high contrast mode.

## Screenshots

High contrast mode off:
<img width="289" alt="app screenshot" src="https://user-images.githubusercontent.com/55459788/100160628-d0d29100-2e64-11eb-9bcf-3f6ccbc7bbc3.PNG">

High contrast mode on:
<img width="279" alt="high contrast app screenshot" src="https://user-images.githubusercontent.com/55459788/100160674-e47df780-2e64-11eb-84ef-51771accb009.PNG">

## Context (please complete the following information)

- OS Name & Version: Windows 10 build 19042.630
- AI-Web Version & Environment: 2020.1123.2 Insider

**Comment by ferBonnin** [comment]
Per conversation with Peter, this is not a bug. It is ok to change the color in the highlighters of the screenshot to match the failure instance color that is next to them in the list of issues
